lib/fake-xhr/index.js

```diff
@@ -115,6 +115,20 @@ function verifyResponseBodyType(body, responseType) {
             );
             error.name = "InvalidBodyException";
         }
+    } else if (responseType === "blob") {
+        if (
+            !isString &&
+            !(body instanceof ArrayBuffer) &&
+            supportsBlob &&
+            !(body instanceof Blob)
+        ) {
+            error = new Error(
```

Comment:
Please add a test for this code.

Reply:
Apologies for the delay on this. I'll add a test case this week so we can get this PR merged.